src/js/content_scripts/input-mapper.js

```diff
@@ -150,8 +150,11 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
                 upwardScroll: null,
                 downwardScroll: null,
                 leftScroll: null,
-                rightScroll: null
-            }
+                rightScroll: null,
+                forwardTab: null,
+                reverseTab: null
+            },
+            currentTabIndex: 0
```

Comment:
We may have to leave it for now, but I'm concerned that your approach will have problems with dynamic interfaces, i.e. if the index is stored and the number of tabbable elements changes.  We can add tests to confirm this down the road.

Reply:
This index will be used only under limited conditions. I guess you're thinking that we are using the index every time we tab, but that's not the case. It is kept as a [fallback option](https://github.com/fluid-lab/gamepad-navigator/pull/10/files#diff-cc20e237566e63150ac1d82f0dbee3f8R249) and will be used only if `indexOf` fails under certain circumstances. However, we can confirm this with tests later as we proceed.